x-pack/plugins/observability_solution/infra/public/components/logs_deprecation_callout.tsx

```diff
@@ -18,14 +18,38 @@ import { css } from '@emotion/css';
 import { SharePublicStart } from '@kbn/share-plugin/public/plugin';
 import { useKibanaContextForPlugin } from '../hooks/use_kibana';
 
-const DISMISSAL_STORAGE_KEY = 'log_stream_deprecation_callout_dismissed';
+const pageConfigurations = {
+  stream: {
+    dismissalStorageKey: 'log_stream_deprecation_callout_dismissed',
+    message: i18n.translate('xpack.infra.logsDeprecationCallout.p.theNewLogsExplorerLabel', {
+      defaultMessage:
+        'The new Logs Explorer makes viewing and inspecting your logs easier with more features, better performance, and more intuitive navigation. We recommend switching to Logs Explorer, as it will replace Logs Stream in a future version.',
+    }),
+  },
+  settings: {
+    dismissalStorageKey: 'log_settings_deprecation_callout_dismissed',
+    message: i18n.translate(
+      'xpack.infra.logsSettingsDeprecationCallout.p.theNewLogsExplorerLabel',
+      {
+        defaultMessage:
+          'The new Logs Explorer makes viewing and inspecting your logs easier with more features, better performance, and more intuitive navigation. These settings will not affect the Logs Explorer behaviour, and is now discouraged configuring them for the legacy Stream application.',
```

Comment:
I think it would work better to start with the settings information to make sure users know right away this page is for logs stream and we don't recommend setting these.

```suggestion
          'These settings only apply to the legacy Logs Stream app, and we do not recommend configuring them. Instead, use Logs Explorer which makes viewing and inspecting your logs easier with more features, better performance, and more intuitive navigation.',
```